pkg/scheduler/api/types.go

```diff
@@ -168,3 +168,9 @@ type ReservedNodesFn func()
 
 // VictimTasksFn is the func declaration used to select victim tasks
 type VictimTasksFn func() []*TaskInfo
+
+// QueueOverusedFn is the func declaration used to judge whether queue is overused.If overused, whether it is all dimensions overused.
```

Comment:
What does `If overused, whether it is all dimensions overused.` mean?

Reply:
Considering queue.deserved = {cpu:1, memory:1, gpu:1}, queue.allocated={cpu:2, memory:2, gpu:1}, so this queue is overused, but not overused at all dimensions.